code/src/components/Checkbox.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import '../style/checkbox.css'
+
+export const Checkbox = ({ musicGenre, setMusicGenre, askQuestion, musicGenreArray }) => {
+
+  return (
+    <div className="question-container">
+      <h2>{askQuestion}</h2>
+      {musicGenreArray.map(item => (
+
+        <label key={item} className="checkboxes" htmlFor={item}>{item}
+          <input 
+            id={item}
+            type="checkbox"
+            value={item}
+            checked={musicGenre}
+            onChange={event => setMusicGenre(event.target.checked)}
+          />
+        </label>
```

Comment:
I think the thing that makes checkbox harder than radio input is because chatbox allows users to click on more than one option, which means we need to store and then push/update the array. You can check again Maks' lecture where he showed us how to create a checkbox of different colors: https://technigo.wistia.com/medias/10nflrshdi.

Reply:
Cool, thanks for this. I will re-watch it